qa.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6.0"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "linkeddata"
```

Comment:
I think it would be a good development dependency.  Then also move the `require 'linkeddata'` to the `spec_helper.rb`